frb_example/pure_dart/dart/lib/ffi.io.dart

```diff
@@ -1,11 +1,13 @@
 import 'dart:ffi';
 import 'dart:io';
 
+import 'package:flutter_rust_bridge/flutter_rust_bridge.dart';
+
 import 'bridge_generated.io.dart';
 export 'bridge_generated.io.dart';
 
 FlutterRustBridgeExampleSingleBlockTestImpl initializeExternalLibrary(String path) {
   return FlutterRustBridgeExampleSingleBlockTestImpl(
-    Platform.isMacOS || Platform.isIOS ? DynamicLibrary.executable() : DynamicLibrary.open(path),
+    (Platform.isMacOS || Platform.isIOS) && !isSilicon ? DynamicLibrary.executable() : DynamicLibrary.open(path),
```

Comment:
well if I don't have it locally, honestly I can't work without editing manually the files and be careful not to commit them. 😅  and the other PR specifically related #762 is IMHO a bit too complicated compared to the simple on/off isSilicon env var here.

Reply:
Then maybe leave it here, and remove it later before merging 😅

Reply:
you want to remove it ? so all people like me will have to manually edit their files to be able to run codegen ?

Reply:
Still suspect https://github.com/fzyzcjy/flutter_rust_bridge/issues/754#issuecomment-1269308105 is bugged code as @Desdaemon mentions :/ 

What about this: No need for #762, just rename `isSilicon` to `useExecutableDylib`. Simple word renaming, done :)

Reply:
If using `useExecutableDylib` I have to inverse the logic (which is fine), since`isSilicon` is like `useOpenDylib`.
So `useExecutableDylib` or `useOpenDylib` ? :)

Reply:
Oh yes, no need to invert logic, just rename to what you suggest

Reply:
ok done in ba14877